CMakeLists.txt

```diff
@@ -45,6 +45,10 @@ target_include_directories(${PROJECT_NAME}
                            $<BUILD_INTERFACE:${PYBIND_JSON_INCLUDE_DIR}>
                            $<INSTALL_INTERFACE:include>)
 
+set_target_properties(${PROJECT_NAME}
+        PROPERTIES
+        PUBLIC_HEADER "${PYBIND11_JSON_HEADERS}")
```

Comment:
Isn't it redundant with the `PUBLIC_HEADER` argument of the `install` command?

Reply:
My understanding is that `set_target_properties` defines the public headers, and the `install` command is given the destination for public headers.